src/com/facebook/buck/event/listener/AbstractConsoleEventBusListener.java

```diff
@@ -197,7 +197,7 @@ public void setPublicAnnouncements(ImmutableList<String> announcements) {
   protected String formatElapsedTime(long elapsedTimeMs) {
     long minutes = elapsedTimeMs / 60_000L;
     String seconds = TIME_FORMATTER.format(locale, elapsedTimeMs / 1000.0 - (minutes * 60));
-    return String.format(minutes == 0 ? "%s" : "%2$dm%1$s", seconds, minutes);
+    return String.format(minutes == 0 ? "%s" : "%2$dm %1$s", seconds, minutes);
```

Comment:
What people are used to seeing is totally based on who are seeing it :)

intellij, gradle and other build systems typically report "x minutes and y seconds"

This change only makes it slightly more readable instead of tally changing it :)